pgaudit--1.6--1.6.1.sql

```diff
@@ -0,0 +1,32 @@
+-- complain if script is sourced in psql, rather than via CREATE EXTENSION
+\echo Use "CREATE EXTENSION pgaudit" to load this file.\quit
+
+-- Drop old triggers and functions
+drop event trigger if exists pgaudit_ddl_command_end;
+drop function if exists pgaudit_ddl_command_end();
+
+drop event trigger if exists pgaudit_sql_drop;
+drop function if exists pgaudit_sql_drop();
+
+-- Create triggers and functions with security definer and search_path
```

Comment:
I understand the search path, but I currently do not understand why this now needs to be security definer. Wouldn't that make the logs show up as the pgaudit extension creator (so a superuser) instead of the user which did the DDL change?

Reply:
No, because this is now not the user of the DDL statement is determined or how the DDL audit is triggered. These functions just get additional information about the DDL that just ran.

There were no test regressions with this commit so we believe the functionality has not been altered.